### PR TITLE
DVCI-46: Add landing page and update styling

### PR DESCRIPTION
**Pull request by Jammjammjamm**
Screen Shot 2021-03-23 at 11 34 26 AM

Screen Shot 2021-03-23 at 11 34 35 AM

Screen Shot 2021-03-23 at 11 38 12 AM

### CODE REVIEW

File: app/views/patients/_form.html.erb [hunk 1]
```diff
@@ -1,40 +1,56 @@
...
+          <div class="select">
+            <%= form.select :gender, options_for_select(Patient::GENDERS)  %>
+          </div>
+        </div>
+      </div>
 
+      <div class="field">
+        <%= form.label :birth_date, class: 'label' %>
+        <div class="control">
+          <%= form.date_field :birth_date, class: 'input' %>
```
Comment (ssayer) [inline app/views/patients/_form.html.erb:42]: Is there a better field we can use for this? I ran into a lot of problems with a free text field as Ruby/Rails can't always parse dates.

**Review by ssayer — CHANGES_REQUESTED** [review]
I might have just missed it, but is there a link to the patient list anywhere? If not, there should be.

File: app/views/immunizations/_form.html.erb [hunk 2]
```diff
@@ -1,32 +1,50 @@
...
+      </div>
 
-  <div class="field">
-    <%= form.label :occurrence %>
-    <%= form.date_select :occurrence %>
-  </div>
+      <div class="field">
+        <%= form.label :occurrence, class: 'label' %>
+        <div class="control">
+          <%= form.date_field :occurrence, class: 'input' %>
```
Comment (ssayer) [inline app/views/immunizations/_form.html.erb:28]: here too (see previous comment on date_field)

File: app/views/patients/_form.html.erb (hunk 1 above)
Comment (Jammjammjamm) [reply app/views/patients/_form.html.erb:42]: You're using safari?

Comment (ssayer) [reply app/views/patients/_form.html.erb]: Yes

Comment (Jammjammjamm) [reply app/views/patients/_form.html.erb]: Everything other than desktop safari and IE have native date inputs. I added some more guidance and front-end validation for safari. The other option is to use a js datepicker, which I'd prefer to hold off on.

**Review by ssayer — APPROVED** [review]

**Pull request merged**